os/root-on-raid.md

```diff
@@ -0,0 +1,58 @@
+# Setting up Root on Raid
+
+Partitions used in RAID arrays need to have the gpt partition type uuid `be9067b9-ea49-4f15-b4f6-f36f8c9e1818` in order to boot. Because Ignition does not current support modifying existing partitions
```

Comment:
In order to boot?  Not true for data volumes.

While we shouldn't spend a lot of time explaining background concepts like "type UUID", we shouldn't casually use terms that may be obscure to the target audience.  One well-placed sentence of explanation can go a long way.

Let's not emphasize the limitations too early.  The user primarily wants to know how to complete the task.